### PR TITLE
#1549: Drop troublesome node-only packages

### LINKED ISSUE #1549: Intermittent build errors: Error: Cannot find module 'cssnano-preset-default'
ERROR in styles.css
styles.css from Css Minimizer
Error: Cannot find module 'cssnano-preset-default'

### PR DESCRIPTION
**Pull request by fregante**
- Fixes #1549 
- Follows https://github.com/pixiebrix/pixiebrix-extension/pull/939

Testable by using the Data URL brick with examples from 

- https://github.com/jsdom/data-urls
- https://en.wikipedia.org/wiki/Data_URI_scheme

Screen Shot 10

### CODE REVIEW

**Comment by BALEHOK** [comment]
I like changes to `.pre-commit-config.yaml` and `package.json`!)

**Comment by twschiller** [comment]
@fregante Don't spend too much time on the data-urls. The support is experimental and we're trying to figure out still which set of file bricks we need. So we can get this merged in to fix the development issues and create a separate ticket for file handling bricks

File: package.json [hunk 1]
```diff
@@ -139,8 +137,7 @@
     "webext-patterns": "^1.1.1",
     "webext-polyfill-kinda": "^0.1.0",
     "webextension-polyfill-ts": "^0.26.0",
-    "webidl-conversions": "^7.0.0",
-    "whatwg-encoding": "^2.0.0"
+    "whatwg-mimetype": "^3.0.0"
```
Comment (fregante) [inline package.json:140]: Small, compliant, no deps, hopefully no issues

https://bundlephobia.com/package/whatwg-mimetype@3.0.0

File: package.json [hunk 2]
```diff
@@ -40,7 +39,7 @@
     "@reduxjs/toolkit": "^1.6.0",
     "@rjsf/bootstrap-4": "^3.1.0",
     "@rjsf/core": "^3.0.0",
-    "@uipath/robot": "^1.2.5",
+    "@uipath/robot": "1.2.5",
```
Comment (fregante) [inline package.json:42]: I also ensured that this is reflected in the lockfile.

The issue is that they forgot to include the built file in v1.2.6, compare:

https://www.unpkg.com/browse/@uipath/robot@1.2.5/dist/
https://www.unpkg.com/browse/@uipath/robot@1.2.6/dist/

**Review by fregante — COMMENTED** [review]
Likely not full-compliant, but it seems to supports the basics and a bit more. 

Screen Shot 14

The other package I tried was buggy enough that I couldn't get it to pass the base tests from mdn and wiki.

https://github.com/killmenot/parse-data-url/issues/created_by/@me

**Review by twschiller — APPROVED** [review]

**Pull request merged**